Build fails on macOS : date: illegal option -- -

With bazel 0.29, `make build` fails on master (93aef5cdde14c6c68ff6456a7117d4b4c6e018f7) on macOS with:
```
[...]
go: finding github.com/cpuguy83/go-md2man v1.0.4
go: error loading module requirements
DEBUG: /private/var/tmp/_bazel_gnormington/8ca8190ddc76e058023fc72217a03eec/external/bazel_gazelle/internal/go_repository.bzl:147:13: fetch_repo: go: finding golang.org/x/net v0.0.0-20190620200207-3b0461eec859
DEBUG: /private/var/tmp/_bazel_gnormington/8ca8190ddc76e058023fc72217a03eec/external/bazel_gazelle/internal/go_repository.bzl:147:13: fetch_repo: go: finding cloud.google.com/go v0.44.0
INFO: Analyzed target //:cip (51 packages loaded, 6762 targets configured).
INFO: Found 1 target...
ERROR: Process exited with status 1
date: illegal option -- -
usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ...
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]
Target //:cip failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 260.607s, Critical Path: 0.11s
INFO: 0 processes.
FAILED: Build did NOT complete successfully
make: *** [build] Error 1
```

--- a/workspace_status.sh
+++ b/workspace_status.sh
@@ -21,7 +21,7 @@ p_() {
 
 git_commit="$(git rev-parse HEAD)"
 git_desc="$(git describe --always --dirty --long)"
-timestamp_utc_rfc3339=$(date --utc --rfc-3339=seconds)
+timestamp_utc_rfc3339=$(date -u +"%Y-%m-%d %H:%M:%S%z")
 timestamp_utc_date_dashes="${timestamp_utc_rfc3339% *}"
 timestamp_utc_date_no_dashes="${timestamp_utc_date_dashes//-/}"
 image_tag="$git_desc"

